### PR TITLE
Create cl-plus artefact

### PR DESCRIPTION
**Pull request by shawntanzk**
Creating a release product that is cl pcl combined artefact called cl-plus branding under cl name would help it be more widely adopted some issues:
1) seems like running a release has a oort error

running `sh run.sh make prepare_release_fast`

leads to:

```
make[1]: *** [cl.Makefile:51: oort] Error 255
make[1]: Leaving directory '/work/src/ontology'
make: *** [Makefile:229: prepare_release_fast] Error 2
Please remember to update your ODK image from time to time: https://oboacademy.github.io/obook/howto/odk-update/.
```

making cl-plus.owl seems to run fine though

Will investigate

2) cl-plus.owl will be > 100MB which won't allow github to host it

Not sure what the solution is here, thoughts @matentzn? Think this is going to become more and more a problem with transcriptomic types being added

### CODE REVIEW

**Comment by matentzn** [comment]
> cl-plus.owl will be > 100MB which won't allow github to host it

We should, like we do with many other ontologies now, use GitHub releases rather than checking in all of these files. You can ask @anitacaron. Bascially, in this order, do:

1. Attach all CL release files as "binaries" to the next release.
1. Change the CL purl config to point to github releases. See [oba config](https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/oba.yml) for an example.
1. Remove all release artefacts _aside from cl-base.owl and cl.obo_ from git version control (git rm, add to .gitignore)
1. Document that, from now on, you have to attach release files to the release. The new ODK version (which I will either finish this or next week) has even an option to add that to the Makefile, but you can do it manually once).

> make[1]: *** [cl.Makefile:51: oort] Error 255

FFS. This has nothing to do with your PR, as this is only needed for cl-simple. I don't know, assign someone to fix it.. :/

**Comment by shawntanzk** [comment]
> >make[1]: *** [cl.Makefile:51: oort] Error 255
> 
> FFS. This has nothing to do with your PR, as this is only needed for cl-simple. I don't know, assign someone to fix it.. :/

Looks like it was a my computer thing, deleted CL and recloned it and making oort seems to work, will try to release after but I think its fixed

**Comment by shawntanzk** [comment]
All looks good, waiting for a release today before running this as a separate release -> this will avoid confounder of changes from release

**Comment by shawntanzk** [comment]
From discussions with @bvarner-ebi
To ensure a as smooth as possible transition to this release method, we should: 
1) Ensure that in the readme, we point users to use purls rather than raw github files
2) In release notes, we should add a point that we have changed the release and users should make sure they use purls rather than raw github files 
3) @bvarner-ebi will follow instructions in this branch with @shawntanzk supporting to ensure that documentation on how to handle releases is clear, with changes made where needed

We plan to do this on 16/12/22.

**Comment by shawntanzk** [comment]
@dosumis this is the cl-plus release PR 
@anitacaron & @matentzn could you help me check that I didn't make any stupid mistakes please?
I will also write in the release notes how we've changed the release thing and to warn people to use the purl if they arent already.
Timing wise, I assume the redirect of purls go first I assume before this gets merged.

**Review by matentzn — CHANGES_REQUESTED** [review]
Small change, then good to go: Keep cl-base.owl and cl.obo, not cl.owl.

File: src/ontology/cl.Makefile
Comment (anitacaron) [inline src/ontology/cl.Makefile:249]: Add labels and format markdown to the diff file. `--labels True  -f markdown`

**Review by anitacaron — CHANGES_REQUESTED** [review]

**Comment by shawntanzk** [comment]
somehow there is some parsing error when i load the iri for cl.obo,
when I use cl.owl and compare it with cl.obo, diff becomes big (assume that's conversion differences)
So middle-ground, I added cl.owl back into the product we upload to the repo, and use that to compare with iri for cl.owl

**Comment by matentzn** [comment]
The point of having cl.obo in the repo is so we see the git diff as a sanity check during releases (easier to read the diff). _all_ irirs should redirect to the github release (IRIs) and _none_ to raw.github!

**Comment by shawntanzk** [comment]
> The point of having cl.obo in the repo is so we see the git diff as a sanity check during releases (easier to read the diff). _all_ irirs should redirect to the github release (IRIs) and _none_ to raw.github!

I get that, but I am facing this issue when I try to uses http://purl.obolibrary.org/obo/cl.obo in the diff: 

```
robot --catalog catalog-v001.xml diff --labels True -f markdown --left-iri http://purl.obolibrary.org/obo/cl.obo --right ../../cl.obo --output diffs/cl-diff.md
[Fatal Error] :1:50: White spaces are required between publicId and systemId.
org.semanticweb.owlapi.io.UnparsableOntologyException: Problem parsing http://purl.obolibrary.org/obo/cl.obo
Could not parse ontology.  Either a suitable parser could not be found, or parsing failed.  See parser logs below for explanation.
The following parsers were tried:
1) org.semanticweb.owlapi.rdf.rdfxml.parser.RDFXMLParser@6999cd39
2) org.semanticweb.owlapi.owlxml.parser.OWLXMLParser@6c37bd27
3) org.semanticweb.owlapi.functional.parser.OWLFunctionalSyntaxOWLParser@31cb96e1
4) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.RioTurtleDocumentFormatFactory@95fd655c
5) org.semanticweb.owlapi.manchestersyntax.parser.ManchesterOWLSyntaxOntologyParser@cb0f763
6) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.NQuadsDocumentFormatFactory@6f9c39ad
7) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.TrigDocumentFormatFactory@27e81c
8) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.RDFJsonDocumentFormatFactory@cd748dc3
9) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.NTriplesDocumentFormatFactory@937ecd36
10) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.BinaryRDFDocumentFormatFactory@3bf24493
11) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.RDFJsonLDDocumentFormatFactory@dcacc47d
12) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.N3DocumentFormatFactory@9a5
13) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.RioRDFXMLDocumentFormatFactory@69b9a3bc
14) org.semanticweb.owlapi.rio.RioTrixParserFactory$TrixParserImpl : org.semanticweb.owlapi.formats.TrixDocumentFormatFactory@27e82d
15) org.semanticweb.owlapi.rdf.turtle.parser.TurtleOntologyParser@57d0fc89
16) org.semanticweb.owlapi.oboformat.OBOFormatOWLAPIParser@6dcd5639
17) org.semanticweb.owlapi.krss2.parser.KRSS2OWLParser@b18c4
18) org.semanticweb.owlapi.rio.RioParserImpl : org.semanticweb.owlapi.formats.RDFaDocumentFormatFactory@264e8d
```

**Comment by matentzn** [comment]
Why is that command in any way related to this PR? when you run the diff, the ../../cl.owl file is present (from the last release!) even if it is not checked into VC. And the diff goal has to depend on `cl.owl` so it rebuilds it for the pull request anyways?

**Comment by shawntanzk** [comment]
oh yeah, sorry, forogot that it will still build in the local machine and the diff is ran on the local machine - have removed cl.owl from the PR

**Review by anitacaron — APPROVED** [review]

**Comment by shawntanzk** [comment]
argh i forgot i changed branch, have to remake stuff, will make another PR with just release and use that >.<

**Pull request merged**
